test/redeyed-smoke.js

```diff
@@ -53,9 +53,9 @@ test('esprima', function (t) {
 
   readdirp({ root: esprimadir, fileFilter: '*.js' })
     .on('data', function (entry) {
-      
+
       var code = fs.readFileSync(entry.fullPath, 'utf-8')
-        , result = redeyed(code, { Keyword: { 'var': '+:-' } }).code
+        , result = redeyed(code, { Keyword: { 'var': '+:-' } }, { buildAst: true }).code
```

Comment:
not sure why we need `ast` here. Shouldn't surrounding still work as it did without an AST?